app/services/flow/flows/provider_start.rb

```diff
@@ -55,30 +55,44 @@ class ProviderStart < FlowSteps
         check_benefits: {
           path: ->(application) { urls.providers_legal_aid_application_check_benefits_path(application) },
           forward: ->(application) do
+            next_step = :open_banking_consents
             if application.applicant_receives_benefit?
               application.change_state_machine_type('PassportedStateMachine')
             else
               application.change_state_machine_type('NonPassportedStateMachine')
+              next_step = :applicant_employed
             end
-            return :substantive_applications if application.used_delegated_functions?
 
-            application.applicant_receives_benefit? ? :capital_introductions : :open_banking_consents
+            return :substantive_applications if application.used_delegated_functions? && application.passported?
+
+            application.applicant_receives_benefit? ? :capital_introductions : next_step
           end
         },
         substantive_applications: {
           path: ->(application) { urls.providers_legal_aid_application_substantive_application_path(application) },
           forward: ->(application) do
             return :delegated_confirmation unless application.substantive_application?
 
-            application.applicant_receives_benefit? ? :capital_introductions : :open_banking_consents
+            application.applicant_receives_benefit? ? :capital_introductions : :non_passported_client_instructions
           end
         },
         delegated_confirmation: {
           path: ->(application) { urls.providers_legal_aid_application_delegated_confirmation_index_path(application) }
         },
+        applicant_employed: {
+          path: ->(application) { urls.providers_legal_aid_application_applicant_employed_index_path(application) },
+          forward: ->(application) do
+            application.applicant_employed? ? :use_ccms_employed : :open_banking_consents
+          end
+        },
         open_banking_consents: {
           path: ->(application) { urls.providers_legal_aid_application_open_banking_consents_path(application) },
-          forward: ->(application) { application.provider_received_citizen_consent? ? :non_passported_client_instructions : :use_ccms }
+          forward: ->(application) do
+            next_step = :non_passported_client_instructions
+            next_step = :substantive_applications if application.applicant_employed? == false && application.used_delegated_functions?
```

Comment:
could you use `if !application.applicant_employed? && ...`

Reply:
i  said 'false' explicitly because applicant_employed could be nil e.g. if the benefit check result is positive then it wouldn't have employed info  , so i didn't know how else to ignore nil